<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>ControlButtonWidget</class>
    <widget class="QWidget" name="ControlButtonWidget">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>400</width>
                <height>300</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>ControlButtonWidget</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
            <item>
                <widget class="QPushButton" name="accelerateButton">
                    <property name="text">
                        <string>Accelerate</string>
                    </property>
                </widget>
            </item>
            <item>
                <widget class="QPushButton" name="decelerateButton">
                    <property name="text">
                        <string>Decelerate</string>
                    </property>
                </widget>
            </item>
            <item>
                <widget class="QPushButton" name="leftTurnButton">
                    <property name="text">
                        <string>Left Turn</string>
                    </property>
                </widget>
            </item>
            <item>
                <widget class="QPushButton" name="rightTurnButton">
                    <property name="text">
                        <string>Right Turn</string>
                    </property>
                </widget>
            </item>
        </layout>
    </widget>
    <resources/>
    <connections/>
</ui>
